eva/optimizer/memo.py

```diff
@@ -54,12 +56,27 @@ def get_group_by_id(self, group_id: int) -> GroupExpression:
     following functions.
     """
 
+    def _get_table_aliases(self, expr: GroupExpression) -> List[str]:
+        """
+        Collects table aliases of all the children
+        """
+        aliases = []
+        for child_grp_id in expr.children:
+            child_grp = self._groups[child_grp_id]
+            aliases.extend(child_grp.aliases)
+        if (expr.opr.opr_type == OperatorType.LOGICALGET or
+                expr.opr.opr_type == OperatorType.LOGICALQUERYDERIVEDGET):
+            aliases.append(expr.opr.alias)
+
```

Comment:
If my previous comments are valid, can we get the aliases from the operator instead of collecting from the children?